packages/data/tests/unit/adapters/facts/elide-test.ts

```diff
@@ -556,14 +556,17 @@ module('Unit | Adapter | facts/elide', function(hooks) {
     assert.expect(1);
     const adapter: ElideFactsAdapter = this.owner.lookup('adapter:facts/elide');
 
-    let errors = [{ message: 'Bad request' }];
-    Server.post(`${HOST}/graphql`, () => [400, { 'Content-Type': 'application/json' }, JSON.stringify({ errors })]);
+    let response = { message: 'Bad request' };
+    Server.post(`${HOST}/graphql`, () => [400, { 'Content-Type': 'application/json' }, JSON.stringify({ response })]);
 
     try {
       await adapter.fetchDataForRequest(TestRequest);
     } catch ({ errors }) {
-      const responseText = await errors[0].statusText;
-      assert.deepEqual(responseText, errors[0].messages, 'fetchDataForRequest an array of response objects on error');
+      assert.deepEqual(
+        response,
+        errors[0].result.response,
```

Comment:
These two. lines should be swapped as the first argument is the actual value and the second argument is the expected value.